### ISSUE TITLE
Error: Not enough funds to send data

### ISSUE THREAD

**Issue by rastrent** [metaplex-foundation/js#448]
Hey there!

Getting `Error: Not enough funds to send data` when running this command. Using Bundlr Storage. Been working well for awhile not doesn't now. I'm wondering if something on bundlrs end changed maybe how they calculated how much is needed?

I'm running this in devnet as well so not sure if that maybe an issue.
https://devnet.bundlr.network 
https://api.devnet.solana.com 

```
    const File = toMetaplexFile(ImageUrl[0], "TestImage");

    const offChainMetadata = {
      name: "test",
      description: "test",
      image: File,
    };

    const { uri } = await metaplex.nfts().uploadMetadata(offChainMetadata);
```
Running the latest version of the js package `0.17.11` 

Thanks for the insight.

**Comment by lorisleiva** [comment]
Hi there 👋 

They did update their pricing logic slightly but I updated the logic on the SDK in preparation for that moment. Starting from v0.17.11, the new logic should be in place (See https://github.com/metaplex-foundation/js/commit/cad98ec1f58f9e43f1fb52e4dfbddf847c781b7b). I did decrease the default price multiplier since the logic is now a lot more accurate which could be why it's not happy. Maybe try a higher price multiplier on the Bundlr plug-in (like 1.5 for instance) to see if that fixes your issue.

**Comment by rastrent** [comment]
oh nice! Yeah let me see about changing the multiplier and i'll let you know. Gracias

**Comment by rastrent** [comment]
Looks like a price multiplier of 3 seams to be the lowest I can go now for it to work.


```
    const metaplex = Metaplex.make(connection)
      .use(
        bundlrStorage({
          address: BUNDLER_NETWORK,
          providerUrl: SOL_NETWORK_URL,
          timeout: 60000,
          priceMultiplier: 3,
        }),
      );
```

**Comment by lorisleiva** [comment]
Wow that is high haha. Thanks for sharing, I'll take a look at the pricing logic when I can to see what's going on. I'll update you here when I do so you're aware of any pricing changing before they are published to the SDK.

**Comment by myyellowshoe** [comment]
Yeah hehe thought it was pretty high as well. Sounds great, thanks again for all the great work on this!

**Comment by rastrent** [comment]
Yeah hehe thought it was a bit high :D. Sounds great, thanks again for all the great work here!
